src/index.html

```diff
@@ -17,11 +17,417 @@
     />
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
```

Comment:
The stylesheet link should point to a CSS file, not a SCSS file. Ensure that 'styles/main.css' is used instead of 'styles/main.scss'.